src/py/crankshaft/crankshaft/clustering/kmeans.py

```diff
@@ -0,0 +1,17 @@
+from sklearn.cluster import KMeans
+import plpy
+
+def kmeans(query, no_clusters, no_init=20):
```

Comment:
Why are we going with n_init=20? Default seems to be 10 at sklearn: http://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html.

I don't know too much about this but I guess we need to find the trade off between performance and accuracy. Do we know how that value affects both?